### ISSUE TITLE
Wiktor → "dodawanie losowych ilosci czasu przed autoodpowiedziami typu dobranoc"

### ISSUE THREAD

**Issue by MargatroidAlice** [TeMPOraL/alice#158]

**Comment by marsjaninzmarsa** [comment]
Apoco?

**Comment by wprzyb** [comment]
Uncanny valley związany z nieludzkim czasem odpowiedzi, nikt ci nie odpowiada na dobranoc w tej samej sekundzie, w ktorej napisales. Alice z tego co pamiętam ma być w miarę człowiecza w warstwie komunikacyjnej (hehe @TeMPOraL taki dżepetto, co struga sobie pinokia) więc stwierdziłem, że to by było spoko

**Comment by marsjaninzmarsa** [comment]
Ale to bot. Wiesz, boty tak mają. Znaczy ok, nie mam nic przeciwko, po prostu dopóki nie zaczynam jej podrywać mi to nie przeszkadza. :D

**Comment by TeMPOraL** [comment]
Fair point @wprzyb, to się wpisuje szerzej w losowe delaye przy wszystkich wypowiedziach.
